disperser/batcher/encoding_streamer.go

```diff
@@ -142,6 +142,10 @@ func (e *EncodingStreamer) Start(ctx context.Context) error {
 					}
 					if strings.Contains(err.Error(), "too many requests") {
 						e.logger.Warn("encoding request ratelimited", "err", err)
+					} else if strings.Contains(err.Error(), "connection reset by peer") {
```

Comment:
is it downgrade or adding more specific logging?

Reply:
It's downgrading these specific error logs to `WARN` level